security/us-east-1/security-hub/README.md

```diff
@@ -0,0 +1,46 @@
+# AWS Security Hub Configuration
+
+This directory contains the Terraform configuration files for setting up AWS Security Hub in the `us-east-1` region.
+
+## Overview
+
+AWS Security Hub provides a comprehensive view of your high-priority security alerts and compliance status across your AWS accounts. This configuration enables the default security standards: AWS Foundational Security Best Practices v1.0.0 and CIS AWS Foundations Benchmark v1.2.0.
+
+## Configuration
+
+- `config.tf`: This file contains the AWS provider settings and the backend configuration for Terraform.
+- `security_hub.tf`: This file contains the resource configuration for AWS Security Hub.
+
+    - `enable_default_standards`: Specifies whether to enable the default security standards.
+    - `auto_enable_controls`: This property determines whether new security controls are automatically enabled as they become available
+    - `control_finding_generator`: Specifies whether the calling account has consolidated control findings turned on. `SECURITY_CONTROL` or `STANDARD_CONTROL`
```

Comment:
I like the overview but this configuration section doesn't seem to add a lot of value to the layer. For example, the `config.tf` file is quite standard in the ref arch and the info about `security_hub.tf` could go in that same file, as in-line documentation, right next to the parameters of the Security Hub resource (or even linking to the Terraform documentation).

Reply:
ok fixed